api/controllers/readCSV.go

```diff
@@ -0,0 +1,93 @@
+package controllers
+
+import (
+	"encoding/csv"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	animeI "bootCampApi/api/interfaces"
+
+	"github.com/gorilla/mux"
+)
+
+func HealthCheck(w http.ResponseWriter, r *http.Request) {
+	w.WriteHeader(http.StatusOK)
+	fmt.Fprintf(w, "Ok...")
+}
+
+func ReadCSV(w http.ResponseWriter, r *http.Request) {
+	// check params
+	id := mux.Vars(r)["id"]
+	idValue := 0
```

Comment:
To initialize variables with default value, please define them in the following way:
```suggestion
	var idValue int
```